packages/components/src/TimeFormatter/TimeFormatter.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { cleanup } from "@testing-library/react";
+import { TimeFormatter } from ".";
+
+afterEach(cleanup);
+
+it("renders a TimeFormatter", () => {
```

Comment:
@kkimble006 Time and dates often have edge cases and are prone to simple mistakes, which mean they could benefit from unit tests. What do you think about adding some simpler ones now? I could think of:
1) Military Time displays properly
2) Civilian time displays properly
3) `12` AM displays properly in civilian and military.

Reply:
We are not using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString to set the time, based on a CivilTime object. I don't think we need to test this.